src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -105,11 +105,17 @@ private void putTranscriptLinesInDatastore(long lectureId, Document document) {
       Node transcriptNode = transcriptNodes.item(nodeIndex);
       Element transcriptElement = (Element) transcriptNode;
       String lineContent = StringEscapeUtils.unescapeXml(transcriptNode.getTextContent());
-      Float lineStart = Float.parseFloat(transcriptElement.getAttribute(ATTR_START));
-      Float lineDuration = Float.parseFloat(transcriptElement.getAttribute(ATTR_DURATION));
-      Float lineEnd = lineStart.floatValue() + lineDuration.floatValue();
+
+      float lineStartSeconds = Float.parseFloat(transcriptElement.getAttribute(ATTR_START));
+      float lineDurationSeconds = Float.parseFloat(transcriptElement.getAttribute(ATTR_DURATION));
+      // I couldn't find any official way to convert a float seconds to long milliseconds without
+      // losing precision.
+      long lineStartMilliseconds = Math.round(lineStartSeconds * 1000);
```

Comment:
Can you declare 1000 as a constant? Or add an inline comment explaining what it represents?

Reply:
Added

Reply:
Not inline tho, right above.  lmk if i should do more